docker-compose.yaml

```diff
@@ -0,0 +1,29 @@
+services:
+
+  wms-consistence-check: &checker
+    image: georchestra/sdi-consistence-check:latest
+    volumes:
+      - ${PWD}:/app
+      - ${PWD}/.sdichecker:/home/snake/.sdichecker
+    entrypoint:
+      - python
+      - checker.py
+    command: >
+      --mode=WMS
+      --server=https://${FQDN}/geoserver/ows
+      --timeout=${TIMEOUT}
+
+  wfs-consistence-check:
+    <<: *checker
+    command: >
+      --mode=WFS
+      --server=https://${FQDN}/geoserver/ows
+      --timeout=${TIMEOUT}
+
+  csw-consistence-check:
+    <<: *checker
+    command: >
+      --mode=CSW
+      --server=https://${FQDN}/geonetwork/srv/fre/csw
+      --geoserver-to-check=${FQDN}
```

Comment:
`--geoserver-to-check=${FQDN}/geoserver/ows` ?

Reply:
FQDN is defined in `.env` file, so there is only one place to change it, like in terraform-georchestra

Reply:
Ok, we should set the service URL in --geoserver-to-check, but this change the interface, so we might be do this in a major release.

I created a dedicated issue: https://github.com/georchestra/sdi-consistence-check/issues/69